dashboard/dashboard.cfm

```diff
@@ -501,6 +501,13 @@
 			var args = '';
 			var dType = null;
 
+			<cfif Len(application._taffy.settings.csrfToken.cookieName) AND Len(application._taffy.settings.csrfToken.headerName)>
+				var csrfCookie = getCookie('<cfoutput>#application._taffy.settings.csrfToken.cookieName#</cfoutput>');
+				if (csrfCookie) {
+					headers['<cfoutput>#application._taffy.settings.csrfToken.headerName#</cfoutput>'] = csrfCookie;
```

Comment:
Probably not a high risk since the dashboard is usually disabled on production, but these JS variables should be wrapped with `encodeForJavaScript`

Reply:
Regarding to e.g. #359 this would break backward compatibility, because encodeForJavaScript() is CF10+. Maybe JSStringFormat() here?

But because of coming from server side data and demands to cookie/header name I also see no real risk here.

Reply:
@pfreitag I don't suppose there's a way to do feature detection, is there? `isDefined("encodeForJavascript")`? I'd rather avoid version number checking, but I'm thinking that might be the only option.

Reply:
isDefined("encodeForJavascript") => false
isNull("encodeForJavascript" => false
structKeyExists(GetFunctionList(), "encodeForJavascript") => true

Reply:
I'm fine with that. If you're making the change, please prefer encodeForJavascript but fall back to JSStringFormat. 👍